.dockerignore

```
@@ -4,3 +4,4 @@
 appveyor.yml
 build
 circle.yml
+./vndr.log
```

(same comment as for the other one)

```suggestion
vndr.log
```